content/docs/ui/inline-editing.md

```diff
@@ -20,96 +20,67 @@ _Inline Editing_ in Tina refers to editing values directly in the area they appe
 
-> Note that it is important to **use a hook to register a form** instead of an HOC or Render Props component. Depending on the Tina packages you are using, the hook names may differ than those seen in the examples.
+`InlineForm` should wrap the page or component where you want to add inline editing, turning the _page into the form itself_. Let's add an `InlineForm` to an example from the [form documentation](/docs/plugins/forms):
 
-`InlineForm` should wrap the page or component where you want to add inline editing, turning the _page into the form itself_. You can then nest multiple `InlineField` components, a render props-based component that allows you to conditionally display an editing interface (when in edit mode) or the page as it will appear in production.
+```diff
 
-The **rough idea** is like this:
-
-```jsx
-<InlineForm form={formObject}>
-  <InlineField name="name-of-the-field">
-    {({ input, status }) => {
-      if (status === 'active') {
-        // we're in editing mode, show an editable interface
-      } else {
-        // we're not in editing mode, show the production layout
-      }
-    }}
-  </InlineField>
-</InlineForm>
-```
-
-### Example
-
-Let's take a modified version of the simplistic example from the [form documentation](/docs/plugins/forms):
-
-```jsx
 import * as React from React
 import ReactMarkdown from 'react-markdown'
 import { useForm, usePlugin } from 'tinacms'
++import { InlineForm } from 'react-tinacms-inline'
 
 export function Page(props) {
   const [modifiedValues, form] = useForm(props.data)
 
   usePlugin(form)
 
   return (
-    <main>
-      <h1>{modifiedValues.title}</h1>
-      <ReactMarkdown source={modifiedValues.markdownContent}>
-    </main>
++   <InlineForm form={form}>
+      <main>
+        <h1>{modifiedValues.title}</h1>
+        <ReactMarkdown source={modifiedValues.markdownContent}>
+      </main>
++   </InlineForm>
   )
 }
 ```
 
-Using `InlineForm` and `InlineField` from `react-tinacms-inline`, we would rewrite the Page component as follows:
+The Inline Form alone won't change any behavior. To edit on the page, you'll need to add [Inline Fields](/docs/ui/inline-editing#all-inline-fields).
+
+## Using Pre-configured Inline Fields
+
+The `react-tinacms-inline` package provides a set of pre-configured Inline Fields that should **work for many use cases**. These fields provide basic input elements and handle the rendering logic between edit and preview mode.
+
+Let's add some Inline Fields using the previous example:
 
-```tsx
+```diff
 import * as React from React
 import ReactMarkdown from 'react-markdown'
 import { useForm } from 'tinacms'
-import { Wysiwyg } from 'react-tinacms-editor'
-import { InlineForm, InlineField } from 'react-tinacms-inline'
++import { InlineForm, InlineTextField } from 'react-tinacms-inline'
++import { InlineWysiwyg } from 'react-tinacms-editor'
```

Comment:
Since `InlineWysiwyg` isn't a default inline-field component I'm thinking we should remove it from the basic example